packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -759,49 +755,302 @@ class CupertinoActionSheetAction extends StatelessWidget {
   }
 }
 
-class _CupertinoActionSheetCancelButton extends StatefulWidget {
-  const _CupertinoActionSheetCancelButton({
-    this.child,
+// This widget renders the background (pressed or not) of a button, and sends
+// whether the button is pressed to the parent.
```

Comment:
a bit confused about this doc string, will this even send when the button is not pressed?